src/olympia/constants/activity.py

```diff
@@ -119,7 +120,7 @@ class DELETE_VERSION(_LOG):
 class ADD_FILE_TO_VERSION(_LOG):
     id = 19
     action_class = 'add'
-    format = _('File {0.name} added to {version} of {addon}.')
+    format = _('File {file} added to {version} of {addon}.')
```

Comment:
This looks to be a File rather than a UserProfile instance?

Reply:
Yes, but the problem was similar. `{0}` didn't exist, the file argument had already be consumed into `{file}`